clang/include/clang/Driver/Options.td

```diff
@@ -7057,6 +7057,8 @@ def fsycl_help_EQ : Joined<["-"], "fsycl-help=">, Flags<[NoXarchOption]>,
 def fsycl_help : Flag<["-"], "fsycl-help">, Alias<fsycl_help_EQ>,
   Flags<[NoXarchOption]>, AliasArgs<["all"]>,
   HelpText<"Emit help information from all of the offline compilation tools">;
+def fsycl_libdevice_path_EQ : Joined<["-"], "fsycl-libdevice-path=">,
+  Visibility<[ClangOption, CLOption, DXCOption, CC1Option]>, HelpText<"Path to libdevice library">;
```

Comment:
Please add individual test cases for each of the options listed under `Visibility`.

Reply:
Thanks, I realized that most of the Visibility options were unnecessary, I've left only `ClangOption` which should be already covered by the test I've added